docs/tutorials/join-dal-baker/run-dal-node.md

```diff
@@ -18,6 +18,8 @@ For example, this command initializes the DAL node with the address of a local `
    octez-dal-node config init --endpoint http://127.0.0.1:8732 --attester-profiles="$MY_ADDRESS"
    ```
 
+   where ``MY_ADDRESS`` is the account address of the baker, starting with ``tz``.
```

Comment:
Just one backtick for code phrases in Markdown

Reply:
Of course, thanks! Fixed everywhere.